CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Minimum Rust version has been bumped to 1.56.0
 
+### Added
+
+- Option `font.builtin_box_drawing` to control whether built-in font is used for drawing box characters
```

Comment:
I can't tell if there is a max line length for this file. I can wrap this if it's too long.

Reply:
It is, 100 columns. So this is a bit too wide.

```
 - Option `font.builtin_box_drawing` to disable the built-in font for drawing box characters
```

This is < 100.